src/xcode/ENA/ENA/Source/Services/TicketValidation/TicketValidation.swift

```diff
@@ -24,17 +24,38 @@ final class TicketValidation: TicketValidating {
 	func initialize(
 		completion: @escaping (Result<Void, TicketValidationError>) -> Void
 	) {
-		validateIdentityDocumentOfValidationDecorator(
-			urlString: initializationData.serviceIdentity
-		) { [weak self] result in
+		validateServiceIdentityAgainstAllowlist(completion: { [weak self] result in
```

Comment:
```suggestion
		validateServiceIdentityAgainstAllowlist { [weak self] result in
```

Just a stylistic preference of mine to use the trailing closure syntax :)